project/build.gradle.kts

```diff
@@ -3,19 +3,20 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
```

Comment:
I was not expecting 232 changed files given the PR description. What happened? It seems this PR includes changes from the klint PR as well. IMO, this PR should only include the changes required for what you describe in the PR title.

Reply:
This PR includes changes from the other 2 PR's, to avoid conflicts.
Only the last two commits are related to this PR's subject.
Will keep in mind to always create new branches from the master.